### ISSUE TITLE
Open several recipients together

### ISSUE THREAD

**Issue by mariongalley** [ThreeSixtyGiving/grantnav#834]
Could we add the ability to select multiple organisations together?

- Ideally you would select them in the Recipient search, open them on the Org page and then have the option to view the grants altogether in GrantNav
- If we can't work out a way to view them together on the Org page, we should at least include the option to view their grants together in GrantNav
- This also relates to #833

**Comment by robredpath** [comment]
This could be implemented as some kind of "cart" functionality - add a number of organisations to a list, and then view all of their grants in one go. 

Or, we could have a "see all grants for orgs returned by search" button - which could be further refined with the recipient org filter in the grants search

**Comment by michaelwood** [comment]
I think this requires much more design thinking and work that we currently have budget for. Insights also seems like a better comparison tool that we could sign post people to?

**Comment by mariongalley** [comment]
Fair enough. The purpose of this isn't to compare organisations, but rather to view together ones that are likely duplicates.

**Comment by robredpath** [comment]
In the first instance, this would be:
* If there are fewer than X results (maybe 5?), then:
* * A button appears saying "see all grants made by" / "see all grants received by" these organisations
* * Clicking it does a grants search for all of the org IDs in the results OR-ed together in the appropriate role

**Comment by mariongalley** [comment]
Removing pre-requisite for live as there are other higher priorities
